Libraries/xcexecution/Headers/xcexecution/Executor.h

```diff
@@ -55,6 +55,12 @@ class Executor {
         libutil::Filesystem *filesystem,
         pbxbuild::Build::Environment const &buildEnvironment,
         Parameters const &buildParameters) = 0;
+
+protected:
+    std::unordered_map<std::string, std::string> extendedEnvironmentVariables(
```

Comment:
This could be a static function in the `.cpp` file, I think — I don't see anything it needs from an instance.